docs/introduction/getting-started.md

```diff
@@ -91,6 +80,25 @@ Then we can just require A-Frame from our app, perhaps built with Browserify or
 require('aframe');
 ```
 
+<a id="deploy-local"></a>
```

Comment:
This is done automatically. You can use `#using-a-local-server` if you rename to `Using a Local Server`